schemas/user/company_info.schema_v1.0.json

```diff
@@ -0,0 +1,164 @@
+        "Georgian", "German", "Ghanaian", "Gibraltarian", "Greek", "Greenlandic",
+        "Grenadian", "Guamanian", "Guatemalan", "Citizen of Guinea-Bissau", "Guinean",
+        "Guyanese", "Haitian", "Honduran", "Hong Konger", "Hungarian", "Icelandic", "Indian",
+        "Indonesian", "Iranian", "Iraqi", "Irish", "Israeli", "Italian", "Ivorian",
+        "Jamaican", "Japanese", "Jordanian", "Kazakh", "Kenyan", "Kittitian",
+        "Citizen of Kiribati", "Kosovan", "Kuwaiti", "Kyrgyz", "Lao", "Latvian", "Lebanese",
+        "Liberian", "Libyan", "Liechtenstein citizen", "Lithuanian", "Luxembourger", "Macanese",
+        "Macedonian", "Malagasy", "Malawian", "Malaysian", "Maldivian", "Malian", "Maltese",
+        "Marshallese", "Martiniquais", "Mauritanian", "Mauritian", "Mexican", "Micronesian",
+        "Moldovan", "Monegasque", "Mongolian", "Montenegrin", "Montserratian", "Moroccan",
+        "Mosotho", "Mozambican", "Namibian", "Nauruan", "Nepalese", "New Zealander",
+        "Nicaraguan", "Nigerian", "Nigerien", "Niuean", "North Korean", "Northern Irish",
+        "Norwegian", "Omani", "Pakistani", "Palauan", "Palestinian", "Panamanian",
+        "Papua New Guinean", "Paraguayan", "Peruvian", "Pitcairn Islander", "Polish",
+        "Portuguese", "Prydeinig", "Puerto Rican", "Qatari", "Romanian", "Russian", "Rwandan",
+        "Salvadorean", "Sammarinese", "Samoan", "Sao Tomean", "Saudi Arabian", "Scottish",
+        "Senegalese", "Serbian", "Citizen of Seychelles", "Sierra Leonean", "Singaporean",
+        "Slovak", "Slovenian", "Solomon Islander", "Somali", "South African", "South Korean",
+        "South Sudanese", "Spanish", "Sri Lankan", "St Helenian", "St Lucian", "Stateless",
+        "Sudanese", "Surinamese", "Swazi", "Swedish", "Swiss", "Syrian", "Taiwanese", "Tajik",
+        "Tanzanian", "Thai", "Togolese", "Tongan", "Trinidadian", "Tristanian", "Tunisian",
+        "Turkish", "Turkmen", "Turks and Caicos Islander", "Tuvaluan", "Ugandan", "Ukrainian",
+        "Uruguayan", "Uzbek", "Vatican citizen", "Citizen of Vanuatu", "Venezuelan", "Vietnamese",
+        "Vincentian", "Wallisian", "Welsh", "Yemeni", "Zambian", "Zimbabwean"
+      ]
+    }
+  },
+  "properties": {
+    "legalBusinessName": {
+      "description": "Legal business name",
+      "type": "string",
+      "maxLength": 120,
+      "pattern": "^[a-zA-Z0-9 ]*$"
+    },
+    "businessName": {
+      "description": "Business name- company's operating name if it's different than the legal business name",
+      "type": "string",
+      "maxLength": 120,
+      "pattern": "^[a-zA-Z0-9 ]*$"
+    },
+    "businessRegisteredCountry": {
+      "description": "The country where the business is registered",
+      "type": "string",
+      "$ref": "#/definitions/country"
+    },
+    "registeredBusinessAddress": {
+      "$ref": "#/definitions/address",
+      "description": "Address of the company's registered office"
+    },
+    "registrationNumberIncorporation": {
```

Comment:
What is the difference between? registrationNumberIncorporation, companiesHouseRegistrationNumber isn't companyHouse number a registrationNumberIncorportation specific for UK?

Reply:
registrationNumberIncorporation - is general field for the company id .
In case company registered country is UK so we have the dedicated field - companiesHouseRegistrationNumber (CRN).  
In case the country is US we will have - EIN  - employerIdentificationNumber"

Reply:
Should we remove the specific fields and have it all fall under the generic field?
Otherwise, how do we make it clear those are oneof

Reply:
@BarAzulay - let's do one of businessRegistrationId field . 
should contains idType & idNumber as we have in individual